CRM/EventsExtras/Test/Fabricator/Event.php

```diff
@@ -0,0 +1,46 @@
+<?php
+use CRM_EventsExtras_Test_Fabricator_Base as BaseFabricator;
+
+/**
+ * Class CRM_EventsExtras_Test_Fabricator_Event.
+ */
+class CRM_EventsExtras_Test_Fabricator_Event extends BaseFabricator {
+
+  /**
+   * Entity's name.
+   *
+   * @var string
+   */
+  protected static $entityName = 'Event';
+
+  /**
+   * Array if default parameters to be used to create an event.
+   *
+   * @var array
+   */
+  protected static $defaultParams = [
+    'title'  => 'Event Sample' ,
+    'event_type_id'  => 3 ,
```

Comment:
there is no guarantee that an event type with id = 3 will be there, so you will need to create an event type first and use it instead.

I would change the fabricate method to do that : 

```
public static function fabricate(array $params = []) {
$eventType = self::createEvenType();
$eventTypeId = $eventType['id'];

.. rest of the code
}

private static function createEvenType() {
 // use API to create event type here
}
```